examples/helloworld/deployment.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     metadata:
       labels:
         app: helloworld
+        app_label_image_short_digest: "{{//helloworld:image.short-digest}}"
```

Comment:
You can move substitution examples under `annotations`, which don't have value size limitations.